application/client.core/src/app/environment/components/sidebar/search.manager/directives/item.directive.ts

```diff
@@ -1,30 +1,50 @@
-import { Directive, ChangeDetectorRef, OnInit, OnDestroy, Input, HostBinding, HostListener, NgZone, ViewContainerRef } from '@angular/core';
-import { Subscription, Observable, Subject } from 'rxjs';
+import { Directive, ChangeDetectorRef, OnInit, OnDestroy, Input, HostBinding, HostListener, NgZone, ViewContainerRef, ViewChild } from '@angular/core';
+import { Subscription } from 'rxjs';
 import { Entity } from '../providers/entity';
 import { Provider, ISelectEvent } from '../providers/provider';
+import { CdkDragRelease } from '@angular/cdk/drag-drop';
+import { MatDragDropResetFeatureDirective } from '../../../../directives/material.dragdrop.directive';
+
+import SearchManagerService, { TRequest } from '../service/service';
+
 
 @Directive({
     selector: '[appSidebarSearchManagerItem]',
 })
 
 export class SidebarAppSearchManagerItemDirective implements OnInit, OnDestroy {
 
+    @ViewChild(MatDragDropResetFeatureDirective) accessor; // First
+
```

Comment:
Please remove the unused block. It's trash from my patch. Sorry, I've forgotten to remove it.